skbio/stats/composition.py

```diff
@@ -973,8 +973,8 @@ def ancom(table, grouping,
 
     # Multiple comparisons
     if multiple_comparisons_correction == 'holm-bonferroni':
-        logratio_mat = np.apply_along_axis(_holm_bonferroni,
-                                           1, logratio_mat)
+        logratio_mat = np.vstack([_holm_bonferroni(logratio_mat[i, :])
+                                  for i in range(logratio_mat.shape[0])])
```

Comment:
not sure what object is logratio_mat, but `for i in logratio_mat` should work and be cleaner?